packages/checkbox-group/src/Component.tsx

```diff
@@ -62,6 +63,16 @@ export type CheckboxGroupProps = {
         },
     ) => void;
 
+    /**
+     * Обработчик блюра.
+     */
+    onBlur?: (event: FocusEvent<HTMLDivElement>) => void;
```

Comment:
Без tabIndex событие с инпутов всплывать будет, и в target будет инпут.
Сейчас event.target.value не получится взять

```suggestion
    onBlur?: (event: FocusEvent<HTMLInputElement>) => void;
```

Reply:
Да, ты прав. Исправил